Src/IronPython/Modules/_io.cs

```diff
@@ -54,6 +55,19 @@ public static partial class PythonIOModule {
 
         private static int O_EXCL => RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 0x400 : RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x800 : 0x80;
 
+        [PythonHidden(PlatformsAttribute.PlatformFamily.Windows)]
```

Comment:
Having `PythonHidden` on a `private` seems kind of odd but I guess it doesn't hurt...

Reply:
Yes, it doesn't hurt, but the less verbosity the better. I assume the same applies to `internal`, though I see there are a few `PythonHidden` that are `internal` already in the codebase.

Reply:
Weird, I wonder why we'd have them on `internal` members. Maybe something that was `public` at some point?

Reply:
Look for example at `ByteArray.UnsafeByteList`. It was established 5 years ago as `internal`, but 4 years ago you added `PythonHidden` in #1000. Do you remember why?

Reply:
Can't say I remember, but looking at https://github.com/IronLanguages/ironpython3/pull/693 I changed `Bytes.GetUnsafeByteArray` (which was public) to a `public` property and then in a follow up commit made it `internal`. I guess I forgot to remove the `PythonHidden` when I did this. I'm guessing `ByteArray.UnsafeByteList` is a copy/paste...